### ISSUE TITLE
Not working with notation software that should (in theory) support it

### ISSUE THREAD

**Issue by gdelgi** [MuseScoreFonts/Leland#22]
Okay, okay, I get it... it wasn't designed for Finale. Nevertheless, that is the software I've got for now, the one I know best, my primary tool. As of version 27 (and, presumably, any version after that), it's supposed to support SMuFL fonts. Naturally, I decided to try Leland, assuming that installing the font, putting the JSON file in the appropriate folder for metadata, and renaming it, would do the trick.

We must all have our pleasant fantasies. Attempting to switch the default music font to Leland in Finale's new SMuFL-ready template produces an error window about Finale not supporting changing the font from SMuFL to non-SMuFL. Odd... this is supposed to be SMuFL-compliant, what's the deal?

Alright, fine, says I... I'll load up a non-SMuFL legacy template and try to change it there since Finale is not reading it as SMuFL somehow. The result is attached; namely, much like in days of yore when one tried to use a SMuFL font in Finale, it produces gibberish. Any advice? Is this likely to ever be fixed, or is Finale pretty much being abandoned as NOFP? (i.e. Not Our F****** Problem)

![leland smufl issue](https://user-images.githubusercontent.com/26174064/154823471-bf20d716-9559-470e-bb82-8ea1bc5d8998.png)

**Comment by oktophonie** [comment]
Leland is designed for any software that supports SMuFL and, as such, should work fine in Finale 27 and above. Yes, I should, and will, update the README to say so! I'm not sure why you are accusing us of "abandoning" Finale, or are being so (passive-)aggressive about it.

 On my machine Leland works fine in Finale, e.g.:

<img width="308" alt="image" src="https://user-images.githubusercontent.com/20604032/154833480-080e050d-28f1-42e1-932a-8517f022a865.png">

All I did was create a new score (used a string quartet template in this case), and change to Leland via Document -> Set Default Music Font.

From your description it sounds as you've done all that's necessary to install the font properly. Nevertheless, if Finale is complaining that Leland appears to be a non-SMuFL font, it _sounds_ like it cannot locate the metadata file
so it would at least be worth double-checking that you have it in the right place and with the right name. If there's anything unclear about the installation instructions I'll be happy to improve them.

**Comment by gdelgi** [comment]
i apologize. Finale is so often considered the proverbial red-headed
stepchild, as it were, by many developers and users that I felt the need to
be preemptively defensive. That's on me, and, again, I am quite sorry.

With that out of the way... I placed the JSON file in exactly the Windows
path instructed in your overview. Is that correct? Or does Finale read it
from somewhere else? Failing that, knowing how tetchy Finale can be about
the exact names of fonts in its font dialogue boxes, should I be renaming
the JSON file something more specific?

**Comment by oktophonie** [comment]
I live in Scotland, so have a great fondness for the red-headed. 😄

The json file should be named `Leland.json`, in the folder described. Maybe you could get away with a lower case L but if Finale is as tetchy as you say, it probably isn't worth the risk!

Also, did you obtain the font by downloading a .zip from the [Releases page](https://github.com/MuseScoreFonts/Leland/releases)? I only ask as some have made the mistake of trying to download the file directly (i.e. [here](https://github.com/MuseScoreFonts/Leland/blob/main/leland_metadata.json)) which, though possible, is likely to mess up the contents/format of the file unless one is very careful about the process.

(v0.75 release shortly to follow, which will mention Finale 27+ support in the README)

**Comment by Jojo-Schmitz** [comment]
Wouldn't it make sense if the name of the json file gets changed (from leland-metadata.json` to `Leland.json`) and then change MuseScore to take that too?

**Comment by oktophonie** [comment]
That's what it was originally called. It was suggested (back in issue #1) to change it to match the filename pattern used in Bravura, a change that a certain @Jojo-Schmitz was apparently in support of ;)

**Comment by Jojo-Schmitz** [comment]
No, back then it was called just `metadata.json`, so even worse ;-)

**Comment by Jojo-Schmitz** [comment]
But we seem to be between a rock and a hard place, ease of supporting Dorico or Finale

**Comment by oktophonie** [comment]
Actually not really, because according to the [SMuFL spec](https://w3c.github.io/smufl/latest/specification/font-metadata-locations.html) the json files should be named _fontname_.json. and put in the appropriate directory to be accessible to other software.

The Bravura metadata file is named `bravura_metadata.json` in Dorico's own directory structure as installed (`C:\Program Files\Steinberg\Dorico4\fonts\metadata` on my machine) and I assume that Dorico itself refers so that file there rather than anywhere else (and this is perhaps why it has that name in the Git repo, too), but for it to be available to other SMuFL-compatible software it needs to go to the 'proper place' with the expected filename, i.e. `C:\Program Files\Common Files\SMuFL\Fonts\Bravura\Bravura.json`. (Unless the software has Bravura built in, as MuseScore does, in which case it doesn't matter.)

So, long story short, the only reason to name the file `leland_metadata.json` is to match the Bravura repo. If the inconvenience outweighs that "benefit" (which it surely does) then yes, we may as well rename it to `Leland.json`.

**Comment by Jojo-Schmitz** [comment]
So Bravura violates SMuFL?

**Comment by gdelgi** [comment]
Tangents aside... did as suggested, and now the font works! Lovely!

**Comment by Tantacrul** [comment]
> So Bravura violates SMuFL?

*Groan*
